### ISSUE TITLE
Crashing on slicing object. 

### ISSUE THREAD

**Issue by mberzse1** [supermerill/Slic3r#4419]
### Version
v2.5.59

### Operating system type + version
windows 11

### Behavior
slicing organic shape, jaw, in hammerhead skull model. when i slice it SS crashes. 
noticed that if i dont orient the model so the base is down then it doesnt crash. 

[LTJaw.zip](https://github.com/user-attachments/files/17050238/LTJaw.zip)

**Comment by ExploiTR** [comment]
Yep, Im getting the same issue. When trying to print anything inclined its crashing when slicing.
